src/components/settings/AppearanceSettings.tsx

```diff
@@ -198,7 +199,8 @@ export const AppearanceSettings: FC = () => {
       <Checkbox
         name="showAccountHostname"
         label="Show account hostname"
-        checked={settings.showAccountHostname}
+        checked={settings.showAccountHostname || hasMultipleAccounts(auth)}
+        disabled={hasMultipleAccounts(auth)}
```

Comment:
Why exactly do we need to disable this on multiple accounts?

Reply:
I see, but can't the hostname be the same on some situations?

Reply:
Indeed, we support repeated hostnames now.

Reply:
so single account, feature flag toggles showing the account header
multiple accounts, feature flag does nada

Reply:
Right, but does it make sense for it to be disabled on the situations where it is the same for example?

Reply:
`showAccountHostname` is misleading nowdays.  It really should be `showAccountNameHeader` or something like that as it represents the row containing account name, account avatar, account platform type, etc